js/app.js

```diff
@@ -194,6 +240,7 @@
    *     event, 'light-green'
    */
 
+btn13.addEventListener('click', function(event){setMyLightClass.bind(btn13).call(this, event, 'light-green');});
```

Comment:
This one is asking for the current context. Replace 'btn13' with the current context